crates/optimism/src/evm.rs

```diff
@@ -235,6 +236,8 @@ mod tests {
         Journal<EmptyDB>,
         L1BlockInfo,
     > {
+        use revm::precompile::bn128;
```

Comment:
This was moved from the top level import because when we run `cargo check --no-default-features` the bn128 import will no longer be available since the substrate-bn and matter-labs dependencies are now optional

Reply:
Now obsolete since bn is not optional